elixir/config/runtime.exs

```diff
@@ -236,34 +236,36 @@ if config_env() == :prod do
 
   # Sentry
 
-  api_external_url_host = URI.parse(compile_config!(:api_external_url)).host
-
-  sentry_environment_name =
-    case api_external_url_host do
-      "api.firezone.dev" -> :production
-      "api.firez.one" -> :staging
-      _ -> :unknown
-    end
-
-  sentry_dsn =
-    case api_external_url_host do
-      "api.firezone.dev" ->
-        "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152"
-
-      "api.firez.one" ->
-        "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152"
-
-      _ ->
-        nil
-    end
-
+  # Base Sentry config
   config :sentry,
-    dsn: sentry_dsn,
-    environment_name: sentry_environment_name,
+    dsn:
+      "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152",
+    environment_name: :unknown,
     enable_source_code_context: true,
     root_source_code_paths: [
       Path.join(File.cwd!(), "apps/domain"),
       Path.join(File.cwd!(), "apps/web"),
       Path.join(File.cwd!(), "apps/api")
     ]
+
+  # Environment-specific Sentry overrides
+  if api_external_url = compile_config!(:api_external_url) do
```

Comment:
This refactoring fixes an issue where if `api_external_url` wasn't actually a URL, the `URI.parse` would fail.

Reply:
Don't we want it to fail though?  Is there a scenario where we don't want that set to a URL?

Reply:
It causes tests and local dev to fail:

```
iex(1)> URI.parse("api")
%URI{
  scheme: nil,
  userinfo: nil,
  host: nil,
  port: nil,
  path: "api",
  query: nil,
  fragment: nil
}
```

Reply:
Hm actually, I'm not sure. Let me double-check.

Reply:
Ah, gotcha.

Reply:
Ah yeah. This config is not specified for the `domain` app by default in the dev/test envs.